tikv/kv.go

```diff
@@ -500,6 +500,9 @@ func (s *KVStore) GetTiKVClient() (client Client) {
 // GetMinSafeTS return the minimal safeTS of the storage with given txnScope.
 func (s *KVStore) GetMinSafeTS(txnScope string) uint64 {
 	if val, ok := s.minSafeTS.Load(txnScope); ok {
+		if val.(uint64) == uint64(math.MaxUint64) {
+			return 0
+		}
```

Comment:
The key point is to prevent the `MinSafeTS` from not being updated permanently because of `MaxUint64`. We should handle it from the updater rather than just the getter.

Reply:
The core question is the getter 
- we introduce `PD API` to not execute `go func which for KV request`, resulting in not updating `safeTSMap`. 		 
    - `updateMinSafeTS` relies on  `safeTSMap` which makes sense(because actually, we can call `updateMinSafeTS` to `kvReuqestUpdater`[to indicate func base]).
- And we need to update `minsafeTS` to make sure when API fails we can fall back to the original way which is by kv request.
- But the core question is:  **`updateMinSafeTS` will return `maxUnit64` when there is no kv request.**
   - to resolve this question, we need to regard `maxUnit64` as 0 which means there is an initial state.